packages/interface/src/features/home/components/Glossary.tsx

```diff
@@ -0,0 +1,66 @@
+      title="Applicant"
+    />
+
+    <FAQItem
+      description="Community members who allocate their voting power (often in the form of credits or tokens) to support projects. Their votes influence both the direct contributions and the matching funds distribution via the quadratic funding mechanism."
+      title="Voters"
+    />
+
+    <FAQItem
+      description="The Coordinator are responsible for deploying the MACI smart contracts, initiating polls, tallying the final results of a vote, and finalizing polls by publishing the final results on-chain."
+      title="Coordinator"
+    />
+
+    <FAQItem
+      description="The team or entity responsible for managing the funding round. Organizers define rules, oversee phases, verify applicant eligibility, and ensure transparency throughout the process."
+      title="Organizers"
+    />
+
+    <FAQItem
+      description="is a community-driven funding method where matching funds are distributed based on the number of contributors and the amounts donated. It amplifies projects with broad support by prioritizing many small contributions over a few large ones, ensuring fair and democratic allocation of resources."
+      title="Quadratic Funding"
+    />
+
+    <FAQItem
+      description="During this phase, applicants submit their proposals, including detailed descriptions, goals, and potential impact. Organizers review and approve submissions to ensure they meet the funding round's criteria."
+      title="Application Phase"
+    />
+
+    <FAQItem
+      description="Voters review the approved proposals and allocate their voting power to the projects they support. Quadratic voting principles apply here, where the cost of additional votes increases quadratically (e.g., 1 vote costs 1 credit, 2 votes cost 4 credits, 3 votes cost 9 credits, etc.)."
+      title="Voting Phase"
+    />
+
+    <FAQItem
+      description="The Coordinator processes all the messages, tallies the results and publshes the on-chain proofs "
+      title="Tallying Phase"
+    />
+
+    <FAQItem
+      description="The final distribution of funds is announced, and the matching funds awarded from the funding pool"
+      title="Results Phase"
+    />
+
+    <FAQItem description="Voting power refers to the credits or tokens a voter can use." title="Voting Power" />
+
+    <FAQItem
+      description="Effective Votes represent the weighted impact of the credits spent after applying quadratic calculations, for example if a voter spends 16 credits on one project, the Effective Votes contributed might be 4."
+      title="Effective Votes"
+    />
+
+    <FAQItem
+      description="A conceptual grouping of projects a voter wants to support, allowing comparison and prioritization before finalizing their votes."
```

Comment:
shouldn't we be sticking with ballots and explain what a ballot is?

Reply:
I agree, after changing everything to "vote" the UI is more confusing